locuszoom.app.js

```diff
@@ -437,13 +452,17 @@ LocusZoom.DataSources.prototype.getSource = function(ns) {
     return this.sources[ns];
 };
 
+LocusZoom.DataSources.prototype.removeSource = function(ns) {
+    return this.setSource(ns, null);
+};
+
 LocusZoom.DataSources.prototype.setSources = function(x) {
```

Comment:
Possibly out of scope for this branch, but it looks like `setSources` is only used in the testing layer? Is there a use case outside of testing where this method would be useful? If not I'd recommend it be defined in the test suite directly to keep the core restricted to logic used in the direct execution of the plugin only.